"""
    The version of the XENDIT API: 3.4.2
"""


import sys
import unittest

import xendit
from xendit.balance_and_transaction.model.validation_error import ValidationError


class TestValidationError(unittest.TestCase):
    """ValidationError unit test stubs"""

    def setUp(self):
        pass

    def tearDown(self):
        pass

    def testValidationError(self):
        """Test ValidationError"""
        # FIXME: construct object with mandatory attributes with example values
        # model = ValidationError()  # noqa: E501
        pass


if __name__ == '__main__':
    unittest.main()
